### PR TITLE
[linux-v6.1.y] RT updates from NXP Real-Time-Edge

### PR DESCRIPTION
**Pull request by MrCry0**

### CODE REVIEW

File: ktypes/preempt-rt/preempt-rt.cfg [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+CONFIG_NET_SCH_CBS=y
+CONFIG_NET_SCH_ETF=y
+CONFIG_NET_SCH_TAPRIO=y
+CONFIG_NET_SCH_MQPRIO=y
+CONFIG_NET_SCH_INGRESS=y
+
+# Enable the generic software mailbox
+CONFIG_GENERIC_SOFTWARE_MAILBOX=y
+# Enable virtio performance driver
+CONFIG_VIRTIO_TRANS=y
```
Comment (ricardosalveti) [inline ktypes/preempt-rt/preempt-rt.cfg:23]: These options will also be enabled on x86, but I guess they should all be fine, right?

File: ktypes/avb/avb.cfg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+CONFIG_EXPERT=y
+CONFIG_PREEMPT_RT=y
+CONFIG_AVB_SUPPORT=y
+CONFIG_NET_SWITCHDEV=y
+CONFIG_DEBUG_INFO=y
+CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
+CONFIG_CPU_FREQ_GOV_ONDEMAND=y
+CONFIG_CPU_FREQ_GOV_USERSPACE=y
```
Comment (ricardosalveti) [inline ktypes/avb/avb.cfg:8]: Same for these extra governors.

File: ktypes/preempt-rt/preempt-rt.cfg (hunk 1 above)
Comment (MrCry0) [reply ktypes/preempt-rt/preempt-rt.cfg:23]: @ricardosalveti thanks for noticing it, I'm making these options enabled only for imx machines. We can't rely on these features to correct work in rt for all other machines even if these machines support these features.

Comment (MrCry0) [reply ktypes/preempt-rt/preempt-rt.cfg]: @ricardosalveti Actually, CONFIG_VIRTIO_TRANS is just for use in test tools, not a real feature :) So I removed it at all.
An imx-specific feature is separated.

File: ktypes/avb/avb.cfg [hunk 3]
```diff
@@ -0,0 +1,8 @@
+CONFIG_EXPERT=y
+CONFIG_PREEMPT_RT=y
+CONFIG_AVB_SUPPORT=y
+CONFIG_NET_SWITCHDEV=y
+CONFIG_DEBUG_INFO=y
```
[the remark this answers is not in the document]
Comment (MrCry0) [reply ktypes/avb/avb.cfg]: @ricardosalveti Fixed

File: ktypes/avb/avb.cfg (hunk 2 above)
Comment (MrCry0) [reply ktypes/avb/avb.cfg]: @ricardosalveti fixed

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Comment by MrCry0** [comment]
@ricardosalveti https://github.com/foundriesio/lmp-kernel-cache/pull/152 is for main

**Pull request merged**